src/modules/image/index.ts

```diff
@@ -375,14 +375,18 @@ export class ImageModule {
        */
       height?: number;
       /**
-       * The color of the image.
+       * The color of the image. Must be a color supported by svg. Defaults to a random color.
```

Comment:
```suggestion
       * The color of the image. Must be a color supported by svg.
```

The `@default` already covers the `Defaults to` part.

Reply:
okay